tools/hangar/__snapshots__/error.ts.snap

```diff
@@ -30,7 +30,52 @@ Duration <DURATION>"
 `;
 
 exports[`inflight_stacktraces.test.w 1`] = `
-"fail ┌ inflight_stacktraces.test.wsim » root/env0/test:assert              
+"[ERROR] assert | Error: assertion failed: false
+  --> ../../../examples/tests/error/inflight_stacktraces.test.w:7:3
+  | let bucket = new cloud.Bucket();
+  | 
+  | test \\"assert\\" {
+7 |   assert(false);
+  |   ^
+at <ABSOLUTE>/inflight_stacktraces.test.w:7:3
```

Comment:
This is the same as the error printed below, I don't think we want this duplicated